frontend/src/app/shared/components/blog-item-preview/blog-item-preview.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="blog">
+  <div class="gradient">
+    <div class = "top">
+      <cvz-tag class="narrow">Tag</cvz-tag>
+    </div>
```

Comment:
poti sterge container-ul `div`, cu clasa top, lasa doar `<cvz-tag>` cu ce e scris in el

Reply:
fixed